integration-tests/test/mrf_lp.spec.ts

```diff
@@ -764,6 +764,32 @@ describe('Liquidity Pool Test', async () => {
     )
   })
 
+  it('Should rebalance ERC20 from L2 to L1', async () => {
```

Comment:
can also add a OMGLikeToken test if you want! there are some tests already with L1->L2 rebalancing